### ISSUE TITLE
Incorrect Application Id

### ISSUE THREAD

**Issue by Rohit-RA-2020** [vedantbahel/viteED#6]
App is not running due to incorrect application-Id on Android platform.
It works if application-Id is changed from **com.viteed.viteedapp** to **com.viteed.viteed_app**.

Can Anyone confirm this issue?

**Comment by vedantbahel** [comment]
Yes. Since you pointed out, can you fix and contribute to close this issue?
